vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
I finally figured out what was the quirk was. I just harried up in that time to add this feature to at least to repo and build used in my projects so I just known nothing about trash in that time, was  enough inattentive so did not realize that shellwords-go is already in upstream repo so just downloaded and put it in my repo somehow did not realizing it is already there. So I put it from master, not from v1.0.0. I am sorry again for so long consuion and taking your time. Now I removed this change in this PR